presto-hive/src/main/java/com/facebook/presto/hive/s3/HiveS3Config.java

```diff
@@ -37,7 +37,7 @@
     private String s3Endpoint;
     private PrestoS3SignerType s3SignerType;
     private boolean s3PathStyleAccess;
-    private boolean s3UseInstanceCredentials = true;
```

Comment:
am not sure about changing the default value. @highker what do you think?

Reply:
This is needed as IAM role configuration will not take effect with the current flow for creating AWS cred provider `PrestoS3FileSystem.createAwsCredentialsProvider` 

```
Optional<AWSCredentials> credentials = getAwsCredentials(uri, conf);
        if (credentials.isPresent()) {
            return new AWSStaticCredentialsProvider(credentials.get());
        }
        if (useInstanceCredentials) {
            return InstanceProfileCredentialsProvider.getInstance();
        }

        if (s3IamRole != null) {
            return new STSAssumeRoleSessionCredentialsProvider.Builder(s3IamRole, "presto-" + uri).build();
        }


```

Reply:
get it.